docs/content.zh/docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -121,7 +121,7 @@ The *job artifacts* are included into the class path of Flink's JVM process with
 * all other necessary dependencies or resources, not included into Flink.
 
 To deploy a cluster for a single job with Docker, you need to
-* make *job artifacts* available locally in all containers under `/opt/flink/usrlib`,
+* make *job artifacts* available locally in all containers under `/opt/flink/usrlib` or pass jar path by *jar-file* argument. 
```

Comment:
or pass jar path by *jar-file* argument -> or pass a list of jars via the `--jars` argument